cmd/juju/commands/switch_test.go

```diff
@@ -321,14 +312,16 @@ func (s *SwitchSimpleSuite) TestSwitchUnknownCurrentControllerRefreshModelsFails
 }
 
 func (s *SwitchSimpleSuite) TestSettingWhenModelEnvVarSet(c *gc.C) {
-	os.Setenv("JUJU_MODEL", "using-model")
-	_, err := s.run(c, "erewhemos-2")
+	err := os.Setenv("JUJU_MODEL", "using-model")
```

Comment:
Just a thought, should we also accept the previous model and controller as env vars?